app/hotels/src/allHotels/NewAllHotels.js

```diff
@@ -1,24 +1,28 @@
 // @flow strict
 
 import * as React from 'react';
-import { ScrollView } from 'react-native';
+import { View } from 'react-native';
 import { StyleSheet } from '@kiwicom/mobile-shared';
 
 import FilterStripe from '../filter/FilterStripe';
 import NewAllHotelsSearch from './NewAllHotelsSearch';
 
 export default function NewAllHotels() {
   return (
-    <ScrollView contentContainerStyle={styles.scrollViewContainer}>
-      <FilterStripe />
-      <NewAllHotelsSearch />
-    </ScrollView>
+    <View style={styles.container}>
+      <View style={styles.absoluteFill}>
+        <View style={{ zIndex: 1000 }}>
```

Comment:
I think it'd be a good idea to have a list of 'official' zIndex's so that it does not become impossible to track which layer is on top of which. Maybe 5 different zIndex's should be enough. What do you think?

Reply:
(Besides, this one style is inline :P)

Reply:
I know, I was fixing that 🙈 

It turns out we already have this in orbit tokens, so I will use from there